pages/thanks-rider.html

```diff
@@ -8,17 +8,28 @@
     <div class="wrapper single-column offset-top">
         <div class="bannerbox">
             <div class="bannerbox__content">
-                <h1>Congratulations</h1>
-                <p>on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
-                <p>Your ride request reference number is: <b class="uuid">ID not loaded, please try again</b>.</p>
-                <p>Please <strong>keep this reference</strong> in case you need to manage your ride request.</p>
-                <p>To view or manage your ride request, visit our <b><a class="self-service-url" href="../self-service/?type=rider">self-service portal</a></b>.</p>
+                <h1 id="thanks-header">Congratulations</h1>
+                <span id="sorryHeading" hidden style="display:none">We are sorry</span>
+                
+                <p id="introPara">on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
+                <span id="sorryPara" hidden style="display:none">It has not been possible to process your request. Please see the information below for more details. We want to help you, so please contact our phone support if you cannot resolve this issue yourself.</span>
```

Comment:
Ah okay, I see how you're using them. I don't agree that having it in the HTML is more elegant, but I don't think it's worth having a battle over.

If you just want to ship it and move on because of time pressure, then go right ahead.